src/org/apache/lucene/index/DocValuesFieldUpdates.m

```diff
@@ -17,9 +17,9 @@
 
 @implementation OrgApacheLuceneIndexDocValuesFieldUpdates
 
-+ (jint)PAGE_SIZE {
-  return OrgApacheLuceneIndexDocValuesFieldUpdates_PAGE_SIZE;
-}
+//+ (jint)PAGE_SIZE {
```

Comment:
@lukhnos Yes I did run into this compiler error:
```
DocValuesFieldUpdates.h:31:9: Expected selector for Objective-C method
```
Plus it also complains in the `.m` file saying:
```
vm_param.h:98:20: Expanded from macro 'PAGE_SIZE'

/*
 *	File:	vm_param.h
 *     Copyright (c) 2000-2012 Apple Inc. All rights reserved.
*/
#define I386_PGBYTES		4096		/* bytes per 80386 page */
#define	PAGE_SIZE		I386_PGBYTES
```
The easiest solution for me was to comment this out. 
I'm not sure wether it is the right way or not.